src/mumble/VersionCheck.cpp

```diff
@@ -205,3 +215,91 @@ void VersionCheck::fetched(QByteArray a, QUrl url) {
 
 	deleteLater();
 }
+#else // USE_APPIMAGE_UPDATER_BRIDGE
+VersionCheck::VersionCheck(bool autocheck, QObject *p, bool focus) : QObject(p) {
+	Q_UNUSED(focus)
+	m_UpdaterDialog = nullptr;
+	m_Revisioner = new AppImageDeltaRevisioner(/*singleThreaded=*/true,/*parent=*/this);
+	connect(m_Revisioner, &AppImageDeltaRevisioner::updateAvailable,
+		this, &VersionCheck::handleUpdateCheck);
+	connect(m_Revisioner, &AppImageDeltaRevisioner::error,
+		this, &VersionCheck::handleUpdateCheckError);
+
+	if(!autocheck)
+		g.mw->msgBox(QString::fromUtf8("Checking for update."));
```

Comment:
By quickly skimming through the code I don't quite understand why you're only printing this message if `autocheck` is `false`. From my understanding this variable doesn't indicate whether the user should be performed or not :thinking:

Reply:
Besides: The note about the update being checked for should already be printed from `main.cpp` iirc.

Reply:
Done.